oak_functions/loader/src/attestation.rs

```diff
@@ -54,7 +51,7 @@ where
     S: std::future::Future<Output = anyhow::Result<Vec<u8>>> + Send + Sync,
 {
     /// Utility object for decrypting and encrypting messages using a Diffie-Hellman session key.
-    encryptor: AeadEncryptor,
+    attested_state_machine: AttestationStateMachine<AttestedState>,
```

Comment:
Again, I think just having an instance of `AttestedState` should work.

Reply:
Now it's an `AttestedSession`